tests/test_rest_api.py

```diff
@@ -102,8 +160,36 @@ def test_recommendation_api_endpoint(_mock_object, client):
     assert jsn['external_request_id'] is not None
 
 
+class FakeRequest:
```

Comment:
You don't need to create an object to mock data. Just use dummy json and can use @mock.patch(<your class name.function name> with that dummy json